playground/eShopLite/AppHost/Properties/launchSettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "profiles": {
-    "http": {
+    "http-resources": {
```

Comment:
Why that change?

Reply:
Just to allow testing in various modes. I'm reverting this.